### ISSUE TITLE
Instruction on testing POST requests from the Console

### ISSUE THREAD

**Issue by mpmlj** [mulesoft/api-workbench#112]
Hi,

Trying to test sending POST requests using my own field values.

My RAML:

```yaml 
#%RAML 1.0
title: My API
version: v1
baseUri: http://127.0.0.1:9001/api/v1
schemas:
  - auth_request: |
      {
        "$schema": "http://json-schema.org/draft-03/schema",
        "type": "object",
        "description": "Auth request",
        "properties": {
            "email": { "type": "string", "required": true},
            "password":  { "type": "string", "required": true }
        }
      }
  - auth_response: |
      {
        "$schema": "http://json-schema.org/draft-03/schema",
        "type": "object",
        "description": "Auth response",
        "properties": {
            "data": { "type": "string", "required": true},
            "err":  { "type": "string", "required": true },
            "status":  { "type": "string", "required": true }
        }
      }
/users/auth:
  post:
    description: Provide user authentication
    body:
      application/json:
        schema: auth_request
    responses:
      200:
        body:
          application/json:
            schema: auth_response
```

However my console has no fields for parameters similar to those appearing with queryParameters.

<img width="560" alt="screen shot 2016-01-28 at 10 07 31 pm" src="https://cloud.githubusercontent.com/assets/6718777/12666117/a87de784-c60b-11e5-9b96-1d4aa821c7d5.png">

When trying to use a JSON string in the field above the "Post" button it does not work, however works from a terminal, as expected. Maybe I'm wrong - could not find documentation on how to use it, if I can store values, maybe it can be auto-populated from an example, etc.

Please advise!

P.S.
This ^ example is for v1.0.
If changing RAML version to 0.8, Console fails: instead of a form, I get an error message "404 How did you get here? You shouldn't be here! Please _report me immediately_! Reference: t" 
Ironically, "report me immediately" link is not working too.

P.P.S.
Testing this v0.8 file with abao ( https://github.com/cybertk/abao ) works correctly.

Thank you,
D

**Comment by blakeembrey** [comment]
Can you define what "it does not work" means when you press the button? Does no action occur or it doesn't update? I can't replicate the issue locally, but there are other interesting ideas here that should be separate issues (make JSON bodies more interactive, auto-populate body from example).

**Comment by mpmlj** [comment]
@blakeembrey 

_Can you define what "it does not work" means when you press the button? Does no action occur or it doesn't update?_

I was just sure that the Console should extract POST fields from the schema/type mentioned in the body, similar to how Query Parameters are shown (otherwise, how to test?) - that is why used a phrase  "does not work". It does make a call, no problem.

_... but there are other interesting ideas here that should be separate issues (make JSON bodies more interactive, auto-populate body from example)._

Right. Even with fields exposed in the Console, it would be great to avoid entering the post params manually and pre-populate them from JSON examples (maybe optional setting). Our QA guys could just open and validate if required.

Thank you

**Comment by blakeembrey** [comment]
Ok, thanks. I understand the issue now. I don't think I'll be able to get around to the JSON schema idea, it's actually _a lot_ of work to make that happen. However, example population is straight forward and should be included. I'll include a button to populate the example into the body.

**Comment by mpmlj** [comment]
@blakeembrey 
Perfect! Thanks a lot.

**Comment by dreamflyer** [comment]
will be published in v0.8.38.
